### PR TITLE
fix: add hiring notice to README

### PR DESCRIPTION
**Pull request by vinckr**
Adds a notice that Ory is hiring to the top of README. 
When this is approved, I would make the same changes to hydra etc. 
As discussed with @BlueAcidFrog

### CODE REVIEW

**Comment by aeneasr** [comment]
Might make sense to set this in meta as a separate section or in the adopters file? then it's synced everywhere

**Comment by vinckr** [comment]
I could create a READMEHEADER.md in the style of ADOPTERS.md, the substitution will take care of the project related links and the others are just the same everywhere. 
Then add a line to sync.sh and 


in every README. 
That should do the trick.
Then we can easily add dockertest and others later on. 
Wdyt?

**Comment by aeneasr** [comment]
SGTM!

**Comment by aeneasr** [comment]
We can merge this anyways, please add the badge to the other repos as well :)

**Comment by vinckr** [comment]
Ok :)

File: README.md [hunk 1]
```diff
@@ -7,7 +7,8 @@
     <a href="https://www.ory.sh/kratos/docs/">Guide</a> |
     <a href="https://www.ory.sh/kratos/docs/sdk/api">API Docs</a> |
     <a href="https://godoc.org/github.com/ory/kratos">Code Docs</a><br/><br/>
-    <a href="https://opencollective.com/ory">Support this project!</a>
+    <a href="https://opencollective.com/ory">Support this project!</a><br/><br/>
+    <a href="https://www.ory.sh/jobs/">Work with us: Ory is hiring!</a>
```
Comment (aeneasr) [inline README.md:11]:
```suggestion
    <a href="https://www.ory.sh/jobs/">Work in Open Source, Ory is hiring!</a>
```

**Comment by aeneasr** [comment]
I updated the description a bit

**Pull request merged**